src/test/java/SmokeTest.java

```diff
@@ -0,0 +1,87 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.*;
+
+import java.io.IOException;
+
+public class SmokeTest extends BaseTest{
+
+    private String emailAddress = randomEmail();
+
+    private String password = randomPassword();
+    private String actualProductName;
+    private double actualPrice;
+    private int orderId;
+    private String productName = "blue jacket";
+    private String thankYouText = "Thank you for your purchase!";
+    private String accountIsCreated = "Thank you for registering with Fake Online Clothing Store.";
+
+    @BeforeTest
+    public void setUp() throws IOException {
+        driver = initializeDriver();
+    }
+
+    @Test
+    public void smokeTest() throws InterruptedException, IOException {
+
+        //open homepage and enter searched word
+        HomePage homePage = new HomePage(driver);
+        homePage.goToHomepage();
+        homePage.enterSearchedWord(productName);
```

Comment:
Before we do any action on a newly loaded page, we should first do an assert to verify that the correct page was loaded. Add a assert to a element that is unique to that webpage.